### ISSUE TITLE
How to improve val_loss

### ISSUE THREAD

**Issue by pasqualeamara** [matterport/Mask_RCNN#1170]
Hello! At the moment, my dataset is composed by approximately 200 images and 400 labels for each class (4 classes).
I tried different data augmentation pipelines (fliplr, flipud, gaussian blur, affine transformation, contrast normalization in particular), to reduce backbone architecture from Res101 to Res50 (without obtaining an improvement), to increase weight_decay (my best result was obtained with weight_decay=0.05) and trying to decrease learning rate every n steps.
Could you suggest me some ways to reduce overfitting (expecially  val_mrcnn_class_loss and  val_mrcnn_mask_loss)?

**Comment by pasqualeamara** [comment]
This is the tb screen of my "best" trainings:

![overfit](https://user-images.githubusercontent.com/44976693/49740763-37207200-fc95-11e8-8223-62380df8fec8.png)

**Comment by fastlater** [comment]
which optimiser are you using? ADAM or SGD? 
Maybe you need more data too.

**Comment by cvtower** [comment]
Hi there,

My project use this repo as the baseline and transfer the coco model as well. According to issues committed, overfitting seems like a main problem here.
My solution：
MS-COCO contains part of my wanted categories, and i believe coco pretrained model was able to recognition samples of these pretrained "class". Then, my training dataset contain 2 parts:
1. annotated samples of other categories beyond coco dataset. In face i totally generate all these samples by my code.
2. VIA annotated samples of all wanted categories.
I mean to add filtered coco dataset as the 3rd part, however, it will take huge GPU hours to train, and i guess the pretrained model should be helpful when training without retrain samples from coco dataset.
At last, my project get pretty nice result.

**Comment by chant0** [comment]
What is the epoch value you set in this result?
